src/Low.ts

```diff
@@ -5,9 +5,9 @@ export interface IAdapter {
   write: (data: any) => Promise<void>
 }
 
-export default class Low {
+export default class Low<T = any> {
   public adapter: IAdapter
-  public data: any
+  public data?: T
```

Comment:
Why `?`?

Reply:
Yes, TS complains otherwise :slightly_smiling_face: 
Since the property doesn't have an initializer and is not assigned in the constructor it is not certain that it is actually assigned, ever.

![lowdb_screenshot](https://user-images.githubusercontent.com/2260305/46324887-133a1380-c5f6-11e8-8d0d-fb6f1c98a584.png)

Reply:
Thanks for the clarification :+1: